### PR TITLE
Modularise migration script, with translations

### PR DESCRIPTION
**Pull request by rwd**

### CODE REVIEW

File: src/actions/translate.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+      for (const name in essence.data) {
+        if (!essence.data[name][fromLocale]) essence.data[name][fromLocale] = '';
+        if (translatedEssence.data[name][toLocale] && (essence.data[name][fromLocale] !== translatedEssence.data[name][toLocale]))
+          essence.data[name][toLocale] = translatedEssence.data[name][toLocale];
+      }
+    }
+  }
+
+  return page;
```
Comment (lbiedinger) [inline src/actions/translate.js]: this returns the page but actually the translations happen on the nested object(s) so this return isn't really necessary.

File: src/support/config.js [hunk 2]
```diff
@@ -30,11 +29,17 @@ const contentfulPreviewClient = contentful.createClient({
...
+turndownService.keep(['cite']);
 
 module.exports = {
-  defaultLocale: 'en-GB',
+  defaultLocale: {
+    contentful: 'en-GB',
+    alchemy: 'en'
+  },
+  maxLengthShort: 255,
+  maxLengthLong: 2000,
```
Comment (lbiedinger) [inline src/support/config.js]: 50000

File: src/actions/translate.js (hunk 1 above)
Comment (rwd) [reply src/actions/translate.js]: But does no harm.

File: src/support/config.js (hunk 2 above)
Comment (rwd) [reply src/support/config.js]: That's applied.

**Pull request merged**